src/app/shared/services/api/account.repo.spec.ts

```diff
@@ -0,0 +1,52 @@
+/* @format */
+import { TestBed, inject } from '@angular/core/testing';
+import {
+  HttpClientTestingModule,
+  HttpTestingController,
+} from '@angular/common/http/testing';
+import { environment } from '@root/environments/environment';
+
+import { HttpService } from '@shared/services/http/http.service';
+import { AccountRepo } from '@shared/services/api/account.repo';
+import { AccountVO } from '@root/app/models';
+
+describe('AccountRepo', () => {
+  let repo: AccountRepo;
+  let httpMock: HttpTestingController;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+      providers: [HttpService],
+    });
+
+    repo = new AccountRepo(TestBed.get(HttpService));
+    httpMock = TestBed.get(HttpTestingController);
+  });
+
+  afterEach(() => {
+    httpMock.verify();
+  });
+
+  it('signUp should call /account/post', () => {
```

Comment:
I wanted to use `shallow-render` for this test, but it appeared that some oddity about our organization of NgModules prevented it from finding my mocked HttpClient (or the HttpTestingController, which I also tried), so I did it in the style of existing API repo tests.